lib/index.js

```diff
@@ -372,6 +394,28 @@ function opt (option) {
   }
 };
 
+/* chan and rmchan sockopt methods. only relevant for subscription sockets */
+Socket.prototype.chan       = function(){
```

Comment:
> After construction, you can call the chan method with only an array.

well `arguments` is certainly array-like. after socket construction `chan()` accepts variable arity, so it's just comma separated parameters